### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by johaan1515**
te he+

### CODE REVIEW

**Comment by Star-Cyber3** [comment]
hi
hi

File: index.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
+<html>
 <head>
-  <h1 poopoo boi is not cool>
-    <h2 he relly isint>
-    <h3 he sucks>
-      <h4 NOOB
+  <h1 poopoo boi is not cool /h1>
```
Comment (Star-Cyber3) [inline index.html:4]:
```suggestion
  <h1>poopoo boi is not cool </h1>
```

**Review by Star-Cyber3 — COMMENTED** [review]
d

**Review by Star-Cyber3 — CHANGES_REQUESTED** [review]
Cant pass

File: index.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
+<html>
 <head>
-  <h1 poopoo boi is not cool>
-    <h2 he relly isint>
-    <h3 he sucks>
-      <h4 NOOB
+  <h1>poopoo boi is not cool </h1>
+    <h2 he relly isint /h2>
```
Comment (Star-Cyber3) [inline index.html:5]:
```suggestion
    <h2> he relly isint </h2>
```

**Review by Star-Cyber3 — CHANGES_REQUESTED** [review]
Click

**Comment by johaan1515** [comment]
E

File: index.html [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
 <head>
-  <h1 poopoo boi is not cool>
-    <h2 he relly isint>
-    <h3 he sucks>
-      <h4 NOOB
+  <h1 poopoo boi is not cool /h1>
+    <h2 he relly isint /h2>
+    <h3 he sucks /h3>
+      <h4 NOOB /h4>
+        <head /h5>
```
Comment (Star-Cyber3) [inline index.html]:
```suggestion
  </head>
  <body>
  <h1> poopoo boi is not cool </h1>
    <h2> he relly isint </h2>
    <h3> he sucks </h3>
      <h4> NOOB </h4>
      </body>
   
```

**Review by Star-Cyber3 — COMMENTED** [review]
NOPE

File: index.html [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
 <head>
-  <h1 poopoo boi is not cool>
-    <h2 he relly isint>
-    <h3 he sucks>
-      <h4 NOOB
+  <h1>poopoo boi is not cool </h1>
+    <h2> he relly isint </h2>
+    <h3. he sucks /h3>
+      <h4> NOOB /h4>
+        <head>
```
Comment (Star-Cyber3) [inline index.html:8]:
```suggestion
    <h3> he sucks </h3>
      <h4> NOOB </h4>
        </body>
```
```suggestion
    <h3. he sucks /h3>
      <h4> NOOB /h4>
        <head>
```

File: index.html [hunk 5]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
+<html>
 <head>
```
Comment (Star-Cyber3) [inline index.html:3]:
```suggestion
<head>
</head>
<body>
```

Comment (Star-Cyber3) [inline index.html:3]:
```suggestion
<head>
</head>
<body>
```

File: index.html [hunk 6]
```diff
@@ -1,6 +1,12 @@
...
-  <h1 poopoo boi is not cool>
-    <h2 he relly isint>
-    <h3 he sucks>
-      <h4 NOOB
+</head>
+<body>
+</head>
```
Comment (Star-Cyber3) [inline index.html:6]:
```suggestion

```

**Review by Star-Cyber3 — APPROVED** [review]
YES

**Pull request merged**